lib/hz2600/Kazoo/Api/Entity/PhoneNumber.php

```diff
@@ -17,10 +17,7 @@ class PhoneNumber extends AbstractEntity
      */
     public function getId() {
```

Comment:
If we are not transforming the ID, we can just remove this function.

Reply:
I can merge this, but since we are doing nothing with the ID now we can just inherit parent::getId() aka just delete this function entirely.

Reply:
I have not had a chance to test that theory yet.  All I know right now the change I first proposed.

Reply:
I will merge for now since it is not working as it currently is, but FYI that should work just fine. getId() is currently (in this PR) calling parent::getId() followed by returning that return value, removing this would use the parent function instead which is more direct.